.gitignore

```diff
@@ -8,6 +8,7 @@
 /reports
 /config/config.json
 /config/config.local.json
+/config.json
```

Comment:
Is this required? I don't think we ever have a config file at the root of the project.
The config file is already in ignored two lines above.